timemachine/cpp/src/kernels/k_log_flat_bottom_bond.cuh

```diff
@@ -0,0 +1,112 @@
+#include "../fixed_point.hpp"
+#include "k_fixed_point.cuh"
+#include "k_flat_bottom_bond.cuh"
+
+// branchless implementation of piecewise function
+template <typename RealType>
+RealType __device__ __forceinline__
+compute_log_flat_bottom_energy(RealType k, RealType r, RealType rmin, RealType rmax, RealType beta) {
```

Comment:
Is this function used? Appears that the kernel calls `compute_flat_bottom_energy`?